src/api/payment-requests/payment-requests.md

```diff
@@ -969,6 +969,102 @@ When you call release on a Pre Auth Payment Request any remaining funds that wer
 | 403    | {% break _ INVALID_PAYMENT_REQUEST_TYPE %} | The Payment Request is not related to a Pre Auth |
 | 403    | {% break _ PRE_AUTH_EXPIRED %}              | `preAuthExpiresAt` has passed                    |
 
+
+<a name="confirm"></a>
+### Make a confirmation against a Pre Auth Payment Request **EXPERIMENTAL**
+
+Confirmations require an `idempotencyKey` in order to ensure that a confirmation will not be processed twice. In the event of a duplicate confirmation request e.g. the payload matches an already completed confirmation, we will return a 200 with the related confirmation Payment Activity, and this confirmation will not be processed again. If an `idempotencyKey` is reused for a different confirmation request, we will return a 403. Note that the `idempotencyKey` only has to be unique for confirmations agains the same Payment Request.
+
+{% reqspec %}
+
+  POST '/api/payment-requests/${paymentRequestId}/confirm'
+  auth 'jwt'
+  path_param 'paymentRequestId', 'MhocUmpxxmgdHjr7DgKoKw'
+  example {
+    title 'Confirm Pre Authorization Payment Request'
+    body ({
```

Comment:
Is it worthwhile to document the optional payload variables that we will allow in future now? Eg: externalRef, purchaseOrderRef